airbyte-workers/src/main/java/io/airbyte/workers/helper/FailureHelper.java

```diff
@@ -20,6 +21,29 @@ public class FailureHelper {
   private static final String JOB_ID_METADATA_KEY = "jobId";
   private static final String ATTEMPT_NUMBER_METADATA_KEY = "attemptNumber";
   private static final String TRACE_MESSAGE_METADATA_KEY = "from_trace_message";
+  private static final String CONNECTOR_COMMAND_METADATA_KEY = "connector_command";
+
+  public enum ConnectorCommand {
+
+    SPEC("spec"),
+    CHECK("check"),
+    DISCOVER("discover"),
+    WRITE("write"),
+    READ("read");
```

Comment:
Setting a string value for these to keep reporting the existing tags the same way to sentry (check, spec, discover)

Reply:
Could add another command for normalize and set that appropriately for the normalizationFailure cases.

Reply:
I intended `connector_command` to map directly to commands that are run against the connectors according to the protocol in order to differentiate more between failures that have an origin of Source or Destination - I think in the case of normalization since the FailureOrigin is always Normalization this shouldn't be necessary (and `normalize` isn't a real connector command)